docs/core/compatibility/aspnet-core/6.0/razor-compiler-doesnt-produce-views-assembly.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Breaking change: Razor: Compiler now produces a single assembly"
```

Comment:
Is there an advantage to using `"`, quotes, in title/description? We never use them. I know it doesn't hurt.

Reply:
You have to use quotes if the text contains colons.